cfy_manager/constants.py

```diff
@@ -39,9 +39,10 @@
 MANAGER_RESOURCES_HOME = join(REST_HOME_DIR, 'resources')
 AGENT_ARCHIVES_PATH = '{0}/packages/agents'.format(MANAGER_RESOURCES_HOME)
 
-USER_CONFIG_PATH = join(CLOUDIFY_HOME_DIR, 'config.yaml')
+CONFIG_FILE_NAME = 'config.yaml'
```

Comment:
Maybe this should be called DEFAULT_USER_CONFIG_NAME?

Reply:
It's not only USER's config file name, DEFAULT_CONFIG_FILE_NAME?

Reply:
fixed in 534f938d